### ISSUE TITLE
cannot use func(s string) string {…} (value of type func(s string) string) as func(strs ...string) string value in assignment

### ISSUE THREAD

**Issue by abdfnx** [elewis787/boa#21]
```
Error: ../../../go/pkg/mod/github.com/elewis787/boa@v0.1.1/cmdlist.go:41:8: cannot use func(s string) string {…} (value of type func(s string) string) as func(strs ...string) string value in assignment
```

I think you need to release a new release for `boa`

**Comment by elewis787** [comment]
I will cut a release today - thanks for circling up on this.

**Comment by elewis787** [comment]
related to #18

**Comment by elewis787** [comment]
resolved in [v0.1.2](https://github.com/elewis787/boa/tree/v0.1.2)

**Comment by abdfnx** [comment]
Thanks @elewis787
